Source/NSPredicateEditor.m

```diff
@@ -47,4 +47,38 @@ - (void) setRowTemplates: (NSArray *)templates
   ASSIGN(_rowTemplates, templates);
 }
 
+- (void) encodeWithCoder: (NSCoder*)aCoder
+{
+  [super encodeWithCoder: aCoder];
+
+  if ([aCoder allowsKeyedCoding])
+    {
+      [aCoder encodeObject: _rowTemplates forKey: @"NSRowTemplates"];
+    }
+  else
+    {
```

Comment:
Else empty?

(Maybe put a comment that this is intentional, if that's worth documenting; as in 'nothing needs to be encoded unless using a keyed coder'. Same below.)

Reply:
This is rather common in GNUstep. For many classes we only have one implementation for (de-)coding, either keyed or the old one. Of course we will both at some point so we leave one if branch empty. I would expect that this is the case for almost one quarter of our classes.